### DISCUSSION TITLE
How to prettify O.Assign result

### DISCUSSION THREAD

**Discussion by mesqueeb** [millsp/ts-toolbelt#203] [Wiki] [answered]
this is my function signature:
```ts
<T extends Record<string, any>, Tn extends Record<string, any>[]>(
  object: T,
  ...otherObjects: Tn
): O.Assign<T, Tn, 'deep'>
```
Currently it gives quite verbose results:
<img width="701" alt="Screenshot 2021-02-26 at 16 27 36" src="https://user-images.githubusercontent.com/3253920/109269281-a28f9380-784f-11eb-80ab-aad2b0cf68c8.png">

Is there a way to prettify the outcome, some kind of wrapper utility I can wrap this with that just creates the new type and shows it similar to this?

![](https://raw.githubusercontent.com/mesqueeb/merge-anything/master/.github/typescript-support.png)

**Comment by millsp** [comment, marked as answer]
Yes, you can use [Object.Compute](https://millsp.github.io/ts-toolbelt/modules/any_compute.html)
